### DISCUSSION TITLE
Print in scale

### DISCUSSION THREAD

**Discussion by SeanOSkea** [hlorus/CAD_Sketcher#137] [Feature Requests]
Really excited about your work, thank you. I know you've said its not a replacement for CAD but the biggest thing that would get it closer would be to be able to export/print in scale. If we can model and dimension in full scale and then render and print or export to a PDF or something in a specific architectural scale, that would be amazing and would make Blender a real tool for physical-world projects. Thanks

**Comment by FunkTank** [comment]
Wholeheartedly agree, would truly change how viable this is as a professional tool.
Also - discussion similar to [114](https://github.com/hlorus/CAD_Sketcher/discussions/114)

**Comment by awschult002** [comment]
just being able to export the sketch as an SVG or other PDF file format would be enough. it doesn't have to have dimensions or constraints drawn yet. request #114 has a great explanation of things, but its a much too lofty goal to start with (imho). 

I really hope to use this tool for laser cutting, and that ultimately means that I need a 2D sketch of whatever I want to cut. Using this 3D tool can help me to visualize the object and set the dimensions for proper interference and everything; but in the end, I just need a 2D svg of the sketch to make my cuts and my plans.

I wonder if blender can export the grease pencil as an svg or something. That might be a quick fix route.

**Comment by awschult002** [comment]
Although what I am going to suggest is not a solution to your overall problem, it might still be helpful advice to limp you along, or get developers looking for more options. Blender has an export utility for similar drawings.  check out freestyle_svg. which will produce sketch, or line images in SVG format.

https://docs.blender.org/manual/en/2.91/addons/render/render_freestyle_svg.html

this will get you line drawings that you can then place into your own PDF. I know that it is more manual than you would like; but perhaps this can provide path to quick solutions on the development side of the house.
